src/models/userApplicationAuthorization.ts

```diff
@@ -0,0 +1,75 @@
+import { ListMeta, Model, ModelList } from '../model'
+import { AuthenticatedResponse, SessionHandler } from '../session'
+
+export enum UserApplicationAuthorizationSorting {
+  ID = 'id',
+  UserId = 'userId'
+}
+
+export interface UserApplicationAuthorizationData {
+  id: number
+  userId: number
+  applicationId: number
+}
+
+export interface UserApplicationAuthorizationResponse extends AuthenticatedResponse {
+  userApplicationAuthorization: UserApplicationAuthorizationData
+}
+
+export interface UserApplicationAuthorizationListResponse extends AuthenticatedResponse {
+  userApplicationAuthorizations: UserApplicationAuthorizationData[]
+  userApplicationAuthorizationsMeta: UserApplicationAuthorizationListResponseMeta
+}
+
+export interface UserApplicationAuthorizationListResponseMeta extends ListMeta {
+  sort: UserApplicationAuthorizationSorting
+}
+
+export class UserApplicationAuthorizations extends ModelList<UserApplicationAuthorization, UserApplicationAuthorizationData, UserApplicationAuthorizationListResponseMeta> {
+  constructor (userApplicationAuthorizations: UserApplicationAuthorizationData[], userApplicationAuthorizationsMeta: UserApplicationAuthorizationListResponseMeta, sessionHandler: SessionHandler) {
+    super(UserApplicationAuthorization, userApplicationAuthorizations, userApplicationAuthorizationsMeta, sessionHandler)
+  }
+}
+
+export class UserApplicationAuthorization extends Model {
```

Comment:
Needs to be 2 different models, one for `user` requests, and one for `developer` requests with actions for their explicit context.